src/config.ts

```diff
@@ -60,6 +61,7 @@ export class Config {
     this._mongo_source_db = process.env.MONGO_SOURCE_DB || "inl-core-m";
 
     this._core_db_connection_string = process.env.CORE_DB_CONNECTION_STRING || "";
+    this._db_name = process.env.DB_NAME || "";
```

Comment:
lets throw a default database name in here - `process.env.DB_NAME || "deep_lynx"`